android/gltfio-android/src/main/java/com/google/android/filament/gltfio/MaterialProvider.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.filament.gltfio;
+
+import com.google.android.filament.Engine;
+
+import java.lang.reflect.Method;
+
+public class MaterialProvider {
+    private long mNativeObject;
+
+    private static Method sEngineGetNativeObject;
+
+    static {
+        System.loadLibrary("gltfio-jni");
```

Comment:
Instead of putting the load in several places you should do what we did in the Filament library and have a single public init() method somewhere that the app must call.

Reply:
Done.